packages/radar/index.d.ts

```diff
@@ -55,7 +67,7 @@ declare module '@nivo/radar' {
         isInteractive?: boolean
         tooltipFormat?: string | CustomFormatter
 
-        legends: LegendProps[]
+        legends?: LegendProps[]
```

Comment:
I couldn't find anything in the docs about whether the legends are optional or not, but there are some examples in the storybook with no legends (https://nivo.rocks/storybook/?path=/story/radar--custom-label-component) and I am already using them without legends in the project I am working on, actually typescript threw me an error because of this one lol.

Reply:
Yes. They are optional.